apps/sscs/sscs-cor-frontend/prod.yaml

```diff
@@ -16,7 +16,4 @@ spec:
         MEDIA_FILES_ALLOWED_ENABLED: "true"
         REQUEST_TAB_ENABLED: "true"
         ALLOW_COOKIE_BANNER_ENABLED: "true"
-        HTTPONLY_COOKIE_FLAG_ENABLED: "true"
```

Comment:
I'd set it to false rather than remove it, because if we do that, it means we still need to investigate issues
we can remove once this is proven to work, and we remove the flag from the code